### PR TITLE
Fix redundant if's for ui_deployment

### PR DESCRIPTION
**Pull request by vasukulkarni**
Unresolved comment from: https://github.com/red-hat-storage/ocs-ci/pull/1122

Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

### CODE REVIEW

File: ocs_ci/deployment/deployment.py [hunk 1]
```diff
@@ -264,17 +264,18 @@ def deploy_ocs_via_operator(self):
...
             logger.info(
                 "Skipping the rest of the deployment because it will be done "
                 "by UI deployment!"
             )
             return
+        else:
+            logger.info("Deployment of OCS via OCS operator")
+            self.label_and_taint_nodes()
+        run_cmd(f"oc create -f {constants.OLM_YAML}")
+        self.create_catalog_source()
```
Comment (petr-balogh) [inline ocs_ci/deployment/deployment.py]: I don't se any extra readability added here by this change and you duplicate the lines from 267 and 268 here.

Comment (vasukulkarni) [reply ocs_ci/deployment/deployment.py]: Line 268 is a function, hence not duplicated its inside *if* , I haven't seen the coding style where the first *if* does nothing but just print info and logical block is handled somewhere down the line with same *if* statement.

Comment (petr-balogh) [reply ocs_ci/deployment/deployment.py]: At least store this cmd: oc create -f {constants.OLM_YAML}
In variable an use on both places if you insist on this change.

Comment (petr-balogh) [reply ocs_ci/deployment/deployment.py]: I updated as you want in my follow up PR for UI deployment here: https://github.com/red-hat-storage/ocs-ci/pull/1130  this pr is still WIP

Comment (vasukulkarni) [reply ocs_ci/deployment/deployment.py]: sure, thanks, will close this one then.

**Pull request closed without merging**